### DISCUSSION TITLE
퍼블리시 에러

### DISCUSSION THREAD

**Discussion by jeejul1010** [naverz/zepeto-studio-kor#91] [5. Q&A]
![image](https://user-images.githubusercontent.com/37315537/146211195-31bbedf8-81bf-4ec2-a0be-47c7e14ef998.png)

계속 이런 에러가 뜨는데 원인을 모르겠네요

**Comment by cheolhui-lee** [comment]
안녕하세요.
혹시 해당 이미지만 봐서는 프로젝트에서 포함하고 있는 텍스쳐의 이슈가 아닐까 하는데요.
유니티를 끄고 프로젝트 폴더 내 Library 폴더를 삭제하시고 다시 유니티를 열어보실 수 있을까요?
